test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +137,37 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            string[] feedlist = GetNetstandrdAndNetcorePackages();
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", feedlist[0]);
+            AddEntry(section, "NetCorePlatforms", feedlist[1]);
```

Comment:
I don't like that the array indexes are special.

How about this?
```C#
DirectoryInfo rootDirectory = GetRepositoryRootDirectory();
DirectoryInfo netStandardLibraryPackageFeed = GetNetStandardLibraryFeed(rootDirectory);
DirectoryInfo netCorePlatformsPackageFeed = GetNetStandardLibraryFeed(rootDirectory);

Assert.True(netStandardLibraryPackageFeed.Exists);
Assert.True(netCorePlatformsPackageFeed.Exists);

…

AddEntry(section, "NetStandardLibrary", netStandardLibraryPackageFeed.FullName);
AddEntry(section, "NetCorePlatforms", netCorePlatformsPackageFeed.FullName);
```

Reply:
Thanks! Changed.